app/src/createExcelFiles/fetchSearches.js

```diff
@@ -0,0 +1,22 @@
+import { SearchItem } from "./modal";
+
+export async function fetchSearches(searchName, searchNumber) {
+  const response = await (
+    await fetch(
+      `https://google-search3.p.rapidapi.com/api/v1/search/q=${searchName}&num=${searchNumber}`,
+      {
+        method: "GET",
+        headers: {
+          "x-rapidapi-key": process.env.REACT_APP_API_KEY,
+          "x-proxy-location": "US",
+        },
+      }
+    )
+  ).json();
+
+  const searchItems = await response.results.map(
```

Comment:
`searchItems` is fine, though usually its better to go with `searchItemList` to specify whether its an array/list, or set, or map, or etc.